### PR TITLE
Fix flux functional test

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

Fix flux functional test.
Flux functional tests are using AAT infra ,  when we put file to container it get processed and result return in AAT queue.
we do not need to add result to queue in AAT.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java [hunk 1]
```diff
@@ -71,9 +74,15 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
         EnvelopeResponse updatedEnvelope = getEnvelope(zipFilename);
         assertThat(updatedEnvelope.getScannableItems()).hasSize(2);
         assertThat(updatedEnvelope.getScannableItems()).allMatch(item -> item.ocrData == null);
-        assertThat(updatedEnvelope.getCcdId()).isEqualTo(ccdId);
+
+        if (fluxFuncTest) {
+            assertThat(updatedEnvelope.getCcdId()).isNotEmpty();
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java:79]: should this not be `EXCEPTION_RECORD` ?

Comment (mdayican) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java:79]: it is the id of whatever ccd creates with our request.

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java [hunk 2]
```diff
@@ -58,10 +58,13 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
         UUID envelopeId = getEnvelope(zipFilename).getId();
 
         // when
-        String ccdId = "ccd-id";
-        String ccdAction = "ccd-action";
-        sendProcessedEnvelopeMessage(envelopeId, ccdId, ccdAction);
-
+        String ccdAction;
```
Comment (doncem) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java:61]: can we use ternary here and send message with if statement?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java [hunk 3]
```diff
@@ -71,9 +74,15 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
...
-        assertThat(updatedEnvelope.getCcdId()).isEqualTo(ccdId);
+
+        if (fluxFuncTest) {
+            assertThat(updatedEnvelope.getCcdId()).isNotEmpty();
+        } else {
+            assertThat(updatedEnvelope.getCcdId()).isEqualTo("ccd-id");
+        }
         assertThat(updatedEnvelope.getEnvelopeCcdAction()).isEqualTo(ccdAction);
 
+
```
Comment (doncem) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java:85]: how this happened?

Comment (mdayican) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java:85]: two different test logic due to infra.

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java (hunk 2 above)
Comment (mdayican) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java:61]: do not see too much point if there will be 'if'

**Pull request merged**